resources/lemmas.md

```diff
@@ -72,6 +82,13 @@ They capture the essential mechanisms used by the two instructions: splitting a
                 : nthbyteof(V, 31, 32)
                 : .WordStack ) => V
       requires 0 <=Int V andBool V <Int pow256
+
+    rule 0 <=Int #asWord(#bufSeg(_, _, _))                 => true
+    rule         #asWord(#bufSeg(_, _, WIDTH)) <Int pow256 => true  requires WIDTH <=Int 32
+
+    rule         #asWord(#bufSeg(_, _, 1)) <=Int 255 => true
```

Comment:
i guess having the side-condition of `WIDTH ==Int 1` is more applicable than directly matching `1`.